scripts/flash/push-time.js

```diff
@@ -3,80 +3,16 @@
-  console.log(chalk.red("SET ==> SET,1518211486 "));
-  console.log(chalk.red("Simply setting system timestamp to 1518211486 "));
-  console.log(chalk.red("                           "));
-  console.log(chalk.red("SET ==> SET,CURRENT "));
-  console.log(chalk.red("Simply setting system timestamp to your current timestamp "));
-  console.log(chalk.red("                           "));
-  console.log(chalk.red("PUSH ===> PUSH,+3d"));
-  console.log(chalk.red("Here we are pushing current timestamp by adding 3 days"));
-  console.log(chalk.red("                           "));
-  console.log(chalk.red("PUSH ===> PUSH,-2w"));
-  console.log(chalk.red("Here we are pushing current timestamp by substracting 2 weeks"));
-  console.log(chalk.red("                           "));
-  console.log(chalk.red("PUSH ===> PUSH,+5s"));
-  console.log(chalk.red("Here we are increasing the current timestamp by 5 seconds"));
-  callback(null);
+function help() {
+  console.log(chalk.red("This method is no longer available, use either"));
+  console.log(chalk.red("push-timestamp"));
+  console.log(chalk.red("set-timestamp"));
 }
 
-function pushTime(augur, params, auth, callback) {
-  var regex = /\d+/g;
-  if (!params || params === "help" || params.split(",").length < 2) {
-    help(callback);
-  } else {
-    var paramArray = params.split(",");
-    var action = paramArray[0];
-    var value = paramArray[1];
-    getTime(augur, auth, function (err, timeResult) {
-      if (err) {
-        console.log(chalk.red(err));
-        return callback(err);
-      }
-      augur.api.Controller.getTimestamp(function (err, timestamp) {
-        if (err) {
-          console.log(chalk.red("Error "), chalk.red(err));
-          return callback(err);
-        }
-        displayTime("current time:", timestamp);
-        if (action === "SET") {
-          if (value === "CURRENT") {
-            value = Math.floor(new Date().getTime()/1000);
-          } else {
-            value = parseInt(value, 10);
-          }
-          setTimestamp(augur, value, timeResult.timeAddress, auth, callback);
-        } else {
-          timestamp = parseInt(timestamp, 10);
-          var modTimeBy = 0;
-          var digit = parseInt(params.match(regex), 10);
-          var subtraction = params.indexOf("-") === -1 ? false : true;
-          var secondsPerDay = 86400;
-          if (params.indexOf("d") !== -1) {
-            modTimeBy = secondsPerDay; // day
-          } else if (params.indexOf("w") !== -1) {
-            modTimeBy = secondsPerDay * 7; // week
-          } else if (params.indexOf("s") !== -1) {
-            modTimeBy = 1; // seconds
-          }
-          var totalMovement = modTimeBy * digit;
-          var newTimestamp = subtraction ? timestamp - totalMovement : timestamp + totalMovement;
-          setTimestamp(augur, newTimestamp, timeResult.timeAddress, auth, callback);
-        }
-      });
-    });
-  }
+function pushTime(augur, args, auth, callback) {
```

Comment:
Are you passing in `augur`, `args`, & `auth` here for consistency? They're not used in this function.

Reply:
yes, that is the pattern. every command gets passed these options.